cmd/config-brancher/main.go

```diff
@@ -160,6 +160,31 @@ func removePeriodics(tests *[]api.TestStepConfiguration) {
 	}
 }
 
+func updatePromotion(currentConfig, futureConfig *api.ReleaseBuildConfiguration, currentRelease, futureRelease, devRelease string) {
+	if futureConfig.PromotionConfiguration == nil {
+		return
+	}
+
+	// we cannot have two configs promoting to the same output, so
+	// we need to make sure the release branch config is disabled
+	futureConfig.PromotionConfiguration.Disabled = futureRelease == devRelease
```

Comment:
You should check that the top-level fields (either `name` or `tag`) are set - if the `determinize-ci-operator` changes I originally suggested are live, no config in the repo should have them set, all should be using `to` - and, in that case, setting the top-level fields is not useful or wanted.

Reply:
I think you might be referring to https://github.com/openshift/ci-tools/pull/3615/files#diff-69f51aa2fa3387ebe31fa93b59fcf08100eb94699b6e84715ecf09dc1f71e2f0 that is not live, not yet at least. I will anticipate the change then 👍🏻